apps/scan/frontend/src/utils/get_page_count.ts

```diff
@@ -0,0 +1,10 @@
+import { printElementToPdf } from '@votingworks/ui';
+import pdfjs from 'pdfjs-dist';
+import { Buffer } from 'buffer';
+import 'pdfjs-dist/build/pdf.worker.entry';
+
+export async function getPageCount(element: JSX.Element): Promise<number> {
+  const pdfBuffer = Buffer.from(await printElementToPdf(element));
+  const pdf = await pdfjs.getDocument(pdfBuffer).promise;
+  return pdf.numPages;
```

Comment:
This is a pretty heavy-weight way to get the page count. Is there no way to tell from the element? Also, I'd prefer to keep pdfjs out of the frontend as I remember it complicating the build and testing a bit. Though maybe not since you didn't change those things?

Reply:
It does complicate testing considerably, but I'm simply not testing it 😄 .

Totally agree on the heavy-weight perspective. If all sub-reports were single paged, we could tell from the element. But because a single sub-report can have multiple pages, we can't actually tell how many pages will be printed until we render to PDF. 

It would be better to get the page count in a way consistent with the `kiosk-browser` API, because right now we're essentially rendering to PDF twice. For example, return the page count after printing. But that would be a bigger refactor, touching multiple repos, that I'd rather not explore for this m17 demonstration.

If this stays in the codebase, I'd support doing this in a smoother way.

Reply:
As for "complicating the build," I'm not sure what you mean, but I haven't seen any problems.